### PR TITLE
Update README.md - Added a Dependencies section, emojis, an issues badge, etc.

### PR DESCRIPTION
**Pull request by tarob0ba**
Thanks for contributing to instascrape! Please review and fill out the checklist below before submitting your request. 

### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### Additional Notes 
Feel free to include any additional information regarding your PR below: 

My pull request was in response to an issue regarding updating the README. My changes were to give extra details and generally improve the README. This is my first contribution to open-source, so hoping that all goes well. Thank you!

### CODE REVIEW

File: README.md
Comment (chris-greening) [inline README.md]: Hey @beckettnormington, thanks so much for contributing, it looks great! 

Before I merge, could you remove just this one section? I figure emphasizing the requirements.txt as opposed to manually installing each dependency is more maintainable as the dependency versions change. Everything else looks great!

**Review by chris-greening — CHANGES_REQUESTED** [review]

File: README.md
Comment (tarob0ba) [reply README.md]: Alright, I believe that I have removed those.

Comment (chris-greening) [reply README.md]: Awesome work! I'll merge right now, congratulations on your first contribution to open source!!!! ❤️

**Pull request merged**
